packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -201,7 +201,8 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     executionPayload: ExecutionPayload,
     versionedHashes?: VersionedHashes,
     parentBlockRoot?: Root,
-    executionRequests?: ExecutionRequests
+    executionRequests?: ExecutionRequests,
+    maxBlobsPerBlock?: number
```

Comment:
`verifyBlocksExecutionPayload` passed in `targetBlobsPerBlock` which is correct as in the [api](https://github.com/ethereum/execution-apis/pull/574/files#diff-18471a795328cf06e000056d594a646c71fa9ce16c35511f3506543b6fc57981R83) but here it's `maxBlobsPerBlock`